src/css/main.scss

```diff
@@ -262,6 +262,28 @@ li {
   font-family: $ff-mono;
 }
 
+.link__toggle {
+  font-size: 1.25rem;
+  line-height: 0;
+  margin-left: .5rem;
+  color: var(--color-davysgrey);
+  cursor: pointer;
+  &:hover {
+    color: var(--color-absolutezero-hover);
+  }
+}
+
+.link__toggle__active {
+  color: var(--color-absolutezero);
+}
+
+.link__toggle__disabled {
```

Comment:
I'm embarrassed to admit I often do BEM backwards, but the rule is to use underscores for the elements and hyphens for the modifiers, so in this case the element is "toggle" and the modifier is "active" or "disabled".
`link__toggle--active` and `link__toggle--disabled`

Reply:
Ah, gotcha. Fixed.